CHANGELOG.rst

```diff
@@ -11,6 +11,9 @@ This document describes changes between each past release.
 - Add an `OpenAPI specification <https://kinto.readthedocs.io/en/latest/api/1.x/openapi.html>`
   for the HTTP API on ``/swagger.json`` (#997)
 
+**New features**
+- ``GET  /admin`` now raises ``307  Temporary Redirect`` to ``/admin/`` with admin plugin.
```

Comment:
I would rephrase this in a less technical manner, like When admin is enabled, `/v1/admin` does not return 404 anymore but now redirects to `/v1/admin/` (trailing slash)